extensions/BMO/lib/Data.pm

```diff
@@ -187,6 +188,27 @@ tie(
   },
 );
 
+# Products that use the triage keyword.
+our @triage_keyword_products = (
```

Comment:
Would be nice if we could add a checkbox to the product details page that enables the triage workflow or not. But we can do that as another bug for later.

Reply:
Given how infrequently a team changes their triage process and new products are added, I didn't think this warranted the overhead of database storage.